server/app.js

```diff
@@ -1,6 +1,7 @@
 import app from "./server.js";
-const PORT = 5080;
 
+const PORT = 8080 || process.env.PORT;
```

Comment:
This will always ignore `process.env.PORT`.

The `x || y` expression will evaluate `x`, and then proceed to evaluate `y` _only if `x` is falsy_. In that case, the entire expression will evaluate to `y`.

In this case, the `8080 || process.env.PORT` will evaluate to 8080, because that number is truthy and the runtime will not proceed to evaulate `process.env.PORT` at all.

If the hosting provider (DigitalOcean?) requires your app to run on port 8080, and that port is configured using the `PORT` environment variable, I would write this line like so:

```js
const PORT = process.env.PORT || 5080;
```

This would mean that it would default to `5080`, but if the `PORT` environment variable is present, it will use that instead, e.g. 8080 for DigitalOcean.

Reply:
Thanks for clarifying, I will keep this in mind in the future.